### ISSUE TITLE
5.0.3 complains about `LogPixelStreaming` in `AVEncoderContext.cpp`

### ISSUE THREAD

**Issue by mattkenefick** [millicast/millicast-publisher-unreal-engine-plugin#22]
I couldn't get my Unreal 5.0.3 on Ubuntu 22.04 to build binaries because of line 319 in `MillicastPublisher/Private/WebRTC/AVEncoderContext.cpp`

```
		// import external memory
		CUresult Result = FCUDAModule::CUDA().cuImportExternalMemory(&mappedExternalMemory, &CudaExtMemHandleDesc);
		if (Result != CUDA_SUCCESS)
		{
			UE_LOG(LogPixelStreaming, Error, TEXT("Failed to import external memory from vulkan error: %d"), Result);
		}
	}
```

I swapped out `LogPixelStreaming` for `LogStreaming` and it worked.

**Comment by dbaldassi** [comment]
I will change this. Should be ``LogMillicastPlayer``

**Comment by dbaldassi** [comment]
Fix in fd961d9f0cf1b288a60d75f10aa4992d00f7e549
